tests/unit/nupic/research/spatial_pooler_boost_test.py

```diff
@@ -339,13 +359,82 @@ def boostTestLoop(self, imp):
     self.boostTestPhase3()
     self.boostTestPhase4()
 
+
   def testBoostingPY(self):
-    self.boostTestLoop("py")
+    self.setUp(imp="py")
+    self.boostTestLoop()
 
 
   def testBoostingCPP(self):
-    self.boostTestLoop("cpp")
-
+    self.setUp(imp="cpp")
+    self.boostTestLoop()
+
+
+  def testBoostingDefaultSPParams(self):
+    """Boosting test with SP's default params."""
+    orig = copy.deepcopy(self.params)
+    params = {}
```

Comment:
I'd like to verify the tests with SP's default params. Just I'm not sure if all of them make sense as some numbers (0.0 syn. inc/dec) are pretty extreme. 

I'm getting

```
======================================================================
FAIL: testBoostingDefaultSPParams (__main__.SpatialPoolerBoostTest)
Boosting test with SP's default params.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unit/nupic/research/spatial_pooler_boost_test.py", line 377, in testBoostingDefaultSPParams
    self.boostTestLoop()
  File "tests/unit/nupic/research/spatial_pooler_boost_test.py", line 354, in boostTestLoop
    self.boostTestPhase1()
  File "tests/unit/nupic/research/spatial_pooler_boost_test.py", line 261, in boostTestPhase1
    self.verifySDRProperties()
  File "tests/unit/nupic/research/spatial_pooler_boost_test.py", line 220, in verifySDRProperties
    "First two SDR's don't overlap enough: \n%r \nvs. \n%r" % (self.lastSDR[0], self.lastSDR[1]))
AssertionError: 1 not greater than 9 : First two SDR's don't overlap enough: 
array([0, 0, 0, ..., 0, 0, 0], dtype=uint32) 
vs. 
array([0, 0, 0, ..., 0, 0, 0], dtype=uint32)
```